app/models/questionnaire.rb

```diff
@@ -0,0 +1,17 @@
+class Questionnaire < ApplicationRecord
+  self.primary_key = "version_id"
+  belongs_to :questionnaire_version, foreign_key: "version_id"
+
+  delegate :localized_layout, :id, :layout, to: :questionnaire_version
```

Comment:
This should not be necessary if we move the localisation logic to the transformers